docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -6,39 +6,42 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/installing-plugins
 
 # Managing Marketplace plugins
 
-The Marketplace is where users can find additional plugins to customize Strapi applications. The Marketplace is located in the admin panel, indicated by ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. In the Marketplace, users can browse or search for plugins, link to detailed plugin descriptions and submit new plugins. This documentation describes how to navigate the Marketplace and how to install Marketplace plugins.
+The Marketplace is where users can find additional plugins to customize Strapi applications, and additional providers to extend plugins. The Marketplace is located in the admin panel, indicated by ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. In the Marketplace, users can browse or search for plugins and providers, link to detailed descriptions for each, and submit new plugins and providers.
```

Comment:
Optionally, you could link here the new "Providers" sub-section you wrote, in case users need more information to understand what providers are.